SPECS/kata-containers/match-Guest-and-Host-cgroup-setup.patch

```diff
@@ -0,0 +1,12 @@
+diff -ruN a/src/runtime/virtcontainers/clh.go b/src/runtime/virtcontainers/clh.go
```

Comment:
Please customize the kernel command line by setting the value of KERNELPARAMS from the spec file, instead of making this code change.

The customized KERNELPARAMS will then make it into configuration-clh.toml a.k.a. configuration.toml, without the need for a code change. See how KERNELPARAMS is used by configuration-clh.in